include/boost/geometry/extensions/gis/projections/impl/pj_init.hpp

```diff
@@ -204,8 +205,14 @@ inline parameters pj_init(R const& arguments, bool use_defaults = true)
 
     if (! s.empty())
     {
-        // TODO: IMPLEMENT SPLIT
-        pin.to_meter = atof(s.c_str());
+        std::size_t const divisor = s.find('/');
+        if (divisor == std::string::npos)
+            pin.to_meter = std::atof(s.c_str());
+        else
+        {
+            std::string const numerator(s.substr(0, divisor)), denominator(s.substr(divisor + 1));
+            pin.to_meter = std::atof(numerator.c_str()) / std::atof(denominator.c_str());
```

Comment:
Ahh, OK, I missed that, thanks. (I also realized that compile-time checks in C++03 are trickier than I thought.) What should happen if either numerator or denominator are zero then? Throw an exception? I see a lot of magic exception numbers thrown throughout the code; are they documented?